Library/Homebrew/formula.rb

```diff
@@ -1364,6 +1364,18 @@ def self.clear_installed_formulae_cache
     @installed = nil
   end
 
+  def self.comparison
```

Comment:
Agreed this should live elsewhere and additionally a more descriptive name would be good (`tap_and_name_comparison` or similar)